changing flow of intro quest

sciorms opened the pull request:
@toddmedema Here is my first pass at issue #750. 

Let me know what feedback you have. I went through the readme to install the local testing suite using yarn but ran into a few errors while running yarn test so I was not able to test this locally.

toddmedema commented on services/app/src/quests/learning_to_adventure.xml:
```diff
@@ -47,15 +47,27 @@
-            <p>Nothing. You'll have to try a different strategy.</p>
+            <choice text="Rolled above 10">
+                <p>{{intimidated = 1}}</p>
+                <p>You successfully intimidate the fairies</p>
```
Perhaps this should go to the combat directly? Seems weird to have it say that intimidation succeeded, and then go back to the "try another option" screen

sciorms replied:
I modified this outcome to flow directly into the subue/force scene as a transition into the arrest scene which then flows into combat.

toddmedema commented on services/app/src/quests/learning_to_adventure.xml:
```diff
@@ -127,6 +133,7 @@
   <combat data-line="153">
     <e>Will o' the Wisp</e>
     <e>Magic Mushroom</e>
+    <e if="intimidated == 1">+2 to all attacks made during this combat.</e>
```
I would explain to the user that this is because their intimidation succeeded

sciorms replied:
I relocated this text to the combat tutorial screen and modified the text to mention that the bonus is because of the intimidation.

sciorms commented:
@toddmedema pushed a new version per your feedback. I changed where the intimidation bonus shows up in the combat text and modified the flow between the market scene and the magic fight scene to be a bit more coherent. Let me know your thoughts.

sciorms commented:
fixes #750
